File: test/func.pas
program func;
　　　var i:integer;
　　　procedure draw_a_line(n:integer); //{该过程打印出连续n 个星号,并换行}
　　　　var j:integer;
　　　　begin
　　　　　for j:=1 to n do
　　　　　　write("*\n");
　　　　end;
　　begin
　　　for i:=1 to 6 do
　　　　draw_a_line(i);{调用过程,第I行打印i个连续星号}
　　end.

File: test/func.c
#include <stdio.h>


int i;

void drawaline(int n) {
    int j;
    for (j = 1; j <= n; j++) 
        printf("* ");
    printf("\n");
}

int main() {
    for (i = 1; i <= 6; i++) 
        drawaline(i);
    return 0;
}